scipy-1.0/covlett.tex

```diff
@@ -0,0 +1,72 @@
+% based on template:
+% https://tex.stackexchange.com/questions/15532/looking-for-cover-letter-template
+
+\documentclass[10pt,stdletter,dateno]{newlfm}
+\usepackage{kpfonts}
+\usepackage{url}
+
+\widowpenalty=1000
+\clubpenalty=1000
+
+\newlfmP{headermarginskip=10pt}
+\newlfmP{sigsize=50pt}
+\newlfmP{dateskipafter=20pt}
+%\newlfmP{addrfromphone}
+%\newlfmP{addrfromemail}
+%\PhrPhone{Phone}
+%\PhrEmail{Email}
+
+\namefrom{Ralf Gommers,
+          Matt Haberland,
+          and Tyler Reddy}
+\addrfrom{%
+    \today\\[10pt]
+    Quansight Labs,\\
+    The Netherlands\\
+    \\
+    BioResource and Agricultural Engineering,\\
+    California Polytechnic State University,\\
+    San Luis Obispo, CA, 93407, USA\\
+    \\
+    CCS-7 Applied Computer Science Group,\\
+    Los Alamos National Laboratory,\\
+    Los Alamos, NM, 87545, USA
+}
+
+\addrto{%
+Prof. Syma Khalid,\\
+School of Chemistry,\\
+University of Southampton,\\
+Highfield Campus,\\
+Southampton,\\
+SO17 1BJ, U.K.}
+
+\greetto{Dear Prof. Khalid,}
+\closeline{Sincerely,}
+\begin{document}
+\begin{newlfm}
+
+We are submitting the enclosed manuscript describing a milestone
+1.0 release of SciPy for consideration to be published in
+\emph{Scientific Reports}. SciPy is the fundamental scientific 
```

Comment:
I find "scientific algorithm computing library" a bit hard to parse. is it:
[ ] scientific (algorithm (computing library))
[ ] scientific ((algorithm computing) library)
[ ] (scientific (algorithm computing)) library
[x] (scientific algorithm) (computing library)
[ ] ((scientific algorithm) computing) library

Reply:
fixed by applying the suggestion below, thanks